content/en/docs/quarks-job/_index.md

```diff
@@ -134,9 +141,6 @@ Each versioned secret has the following characteristics:
   - `quarks.cloudfoundry.org/secret-version` with a value set to the `ordinal` of the secret
 - an annotation of `quarks.cloudfoundry.org/source-description` that contains arbitrary information about the creator of the secret
 
-## See also
```

Comment:
I would really prefer to keep references around... looks more polished, but as you wish :+1: